src/Faithlife.Tracing/NullTraceProvider.cs

```diff
@@ -0,0 +1,18 @@
+namespace Faithlife.Tracing
+{
+	/// <summary>
+	/// An <see cref="ITraceProvider"/> that returns <c>null</c> for <c>CurrentTrace</c>.
+	/// </summary>
+	public sealed class NullTraceProvider : ITraceProvider
+	{
+		/// <summary>
+		/// An instance of <see cref="NullTraceProvider"/>.
+		/// </summary>
+		public static ITraceProvider Instance { get; } = new NullTraceProvider();
```

Comment:
`readonly`

Reply:
This is already a read-only auto-property. Is `public static readonly ITraceProvider Instance = new NullTraceProvider();` preferred?

Reply:
Oh, ha, sorry, I'm used to readonly fields, but this works too.